test/cypress/e2e/register_challenge.spec.cy.js

```diff
@@ -0,0 +1,298 @@
+describe('Register Challenge page', () => {
+  context('desktop', () => {
+    beforeEach(() => {
+      cy.visit('/#/register-challenge');
+      cy.viewport('macbook-16');
+    });
+
+    it('allows user to display help dialog and read all FAQ items', () => {
```

Comment:
Here line 8-102 are the duplicate code tests. Will it be possible to write one basic test (help dialog) and import it here?

Reply:
Based on a previous discussion, I have moved these tests into the component.
Tests which will be share but cannot be moved into component (switching language) are extracted into `commonTests.js`.